sympy/utilities/lambdify.py

```diff
@@ -150,11 +147,12 @@ def _import(module, reload="False"):
     # Add translated names to namespace
     for sympyname, translation in translations.items():
         namespace[sympyname] = namespace[translation]
+    namespace['Abs'] = abs
```

Comment:
I think this line is worthy of an explanatory comment and reference to the issue and this PR.